src/Agent.php

```diff
@@ -264,21 +264,30 @@ public function showForm($id, array $options = [])
             echo "<tr class='tab_bg_1'>";
             echo "<td>" . __('Network discovery threads') . "</td>";
             echo "<td>";
+            $general = __('General setup');
+            if (isset($CFG_GLPI['threads_networkdiscovery'])) {
+                $general = sprintf('%1$s (%2$s)', $general, $CFG_GLPI['threads_networkdiscovery']);
+            }
             Dropdown::showNumber(
                 'threads_networkdiscovery',
                 [
-                'value' => $this->fields['threads_networkdiscovery'],
-                'toadd' => [__('General setup')]
+                    'value' => $this->fields['threads_networkdiscovery'],
+                    'toadd' => [$general],
```

Comment:
General should be bind to 0.
```suggestion
                    'toadd' => [0 => $general],
```

Reply:
This is not strictly needed, it will be the default